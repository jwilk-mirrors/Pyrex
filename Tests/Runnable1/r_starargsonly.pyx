def spam(*args):
	print "Args:", args

