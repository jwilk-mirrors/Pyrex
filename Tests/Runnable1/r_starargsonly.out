Args: ()
Args: (42,)
Args: ('one', 2, 'buckle my shoe')
